server/routes/visitJourney/selectVisitors.ts

```diff
@@ -37,6 +37,13 @@ export default class SelectVisitors {
       formValues.visitors = visitSessionData.visitors.map(visitor => visitor.personId.toString())
     }
 
+    let returnAddress
+    if (isUpdate) {
+      returnAddress = `/visit/${visitSessionData.previousVisitReference}`
+    } else {
+      returnAddress = `/prisoner/${offenderNo}`
+    }
+
```

Comment:
A nicer way to write this would be:

```
const returnAddress = isUpdate ? `/visit/${visitSessionData.previousVisitReference}` : `/prisoner/${offenderNo}`
```

Reply:
Otherwise good 👍